### ISSUE TITLE
PHP CodeIntel does not inherit interface methods

### ISSUE THREAD

**Issue by garak** [Komodo/KomodoEdit#751]
Use case: interface A inherits from interface B.
If I try to automcomplete a method of interface A, everything is fine (e.g. I got that method autocompleted). 
Instead, if I try to autocomplete a method if interface B (that is also in A, of course, by inheritance), it's not autocompleted.

**Comment by mitchell-as** [comment]
Hi, I thought I understood where you were coming from, but when I tried to reproduce on my machine, I couldn't. Would you please post an example? I don't know much PHP. Thanks.

**Comment by garak** [comment]
``` php
<?php

Interface A
{
    public function foo();
}

Interface B extends A
{
    public function bar();
}

class classB
{
    public function __construct(B $b)
    {
        $b->bar();  // this works, it's autocompleted and linked
        $b->foo();  // this should work, but it doesn't
    }
}
```

**Comment by Naatan** [comment]
Reopening this so that it's integrated to 9.3.2, please do not integrate until 9.3.1 is out the door.

**Comment by mitchell-as** [comment]
Merged for 9.3.2 release.
